mailpoet/lib/WooCommerce/TransactionalEmails/Renderer.php

```diff
@@ -70,6 +70,13 @@ public function getHTMLAfterContent() {
    * - We update inlined font-family rules in the content block generated by Woo
    */
   public function updateRenderedContent(NewsletterEntity $newsletter, string $content): string {
+    $isSavedWithStyledWooBlock = $newsletter->getGlobalStyle('woocommerce', 'isSavedWithUpdatedStyles');
+    // For Backward compatibility do not apply styles for content unless the template was edited with the editor
+    // and user visually checked and is aware of updated styles feature.
+    // This check is save to remove after 2025-07-31
```

Comment:
Is there an assumption that everyone will re-save their current Woo templates within a year? As far as I understand, if we remove this check after a year, it can still break the styling of emails that were saved before now but haven't been re-saved since then. Unless there is a need to remove this check, I think it would be safer to keep it so we don't have to debug why something broke in a year's time.

Reply:
> Is there an assumption that everyone will re-save their current Woo templates within a year?

Yep. That's how I was thinking about it. I don't like this type of condition, so I wanted to figure out a way to get rid of it. But you're right. It may cause unexpected issues after a year or so. I'll remove that part of comment.